docs/cudf/source/user_guide/guide-to-udfs.ipynb

```diff
@@ -7,13 +7,24 @@
     "# Overview of User Defined Functions with cuDF"
```

Comment:
<p>While it's good to do this in tests, maybe we don't want to compare the invocations of <code>apply()</code> with Pandas -- unless there's a difference we want to call out.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/10607/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>